spec/models/spree/retail/shopify/generate_pos_order_spec.rb

```diff
@@ -41,7 +41,7 @@
 
     it 'ships the shipments' do
       subject
-      expect(last_order.shipments.state).to eql('shipped')
+      expect(last_order.shipments.first.state).to eql('shipped')
```

Comment:
Does the state-machine make a `shipped?` predicate available on the shipment? Could this be more clearly expressed as `expect(last_order.shipments).to all(be_shipped)`?